app/controllers/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UsersController<ApplicationController
+
+  def show
+    @user = User.find(session[:id])
+  end
+
+  def new
+    @user = User.new
```

Comment:
Creating blank user variable (but notably not saving to DB) for so that view doesn't error out the first pass through. Subsequent calls to this view will come from redirects in Create method & will populate it with previously submitted data.